webpack/assets/javascripts/react_app/components/common/forms/Form.js

```diff
@@ -20,9 +20,9 @@ const Form = ({
 }) => (
   <form className={className} onSubmit={onSubmit}>
     {error && (
-      <Alert className="base in fade" type="danger">
+      <Alert className="base in fade" type={error.severity}>
```

Comment:
Would it be makes sense to handle the default here instead of in the form component? something like
```suggestion
      <Alert className="base in fade" type={error.severity || "danger"}>
```

Reply:
Looks like this change was somehow lost in the rebases?

Reply:
It is back.